### ISSUE TITLE
Low version node module qs cause query params parse bug

### ISSUE THREAD

**Issue by fuqiang915** [balderdashy/sails#3370]
when I set query parms like this:
http://localhost:1339?where[price][][gteq]=6000&where[price][][lteq]=6999

in Controller i print the log of req.parms.all() ,got this:
 {where:{price: [{"lteq":"6999"}]}}
but the correct one is :
 {where: {price: [{"gteq":"6000","lteq":"6999"}]}

I found that the cause of this issue is sails using low version of node module qs  which path is /[project_root]/node_modules/sails/node_modules/connect/node_modules/qs

when i change this version from 1.0.2  to 6.0.0 the problem is solved.
please check this out.
sails version 0.11.0

**Comment by sailsbot** [comment]
Thanks for posting, @fuqiang915.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
